docs/dev/getting-started.md

```diff
@@ -1 +1,317 @@
+    >
+    >         sudo port install py27-pip
+    >
+    > [Homebrew](http://brew.sh/) users on the OS X platform will have
+    > pip already installed
+ 
+```html
+<!-- -->
+```
+b.  Run the install-dependencies maketarget:
+ 
+        make install-dependencies
+ 
+    This will create a virtual python environment to use for Symbulation
+    development. In this environment it will install:
+    [Sphinx](http://sphinx-doc.org/),
+    [Breathe](https://breathe.readthedocs.org/en/latest/), and
+    [doxygen](http://www.stack.nl/~dimitri/doxygen/), packages we use to
+    generate the documentation for Symbulation
+ 
+### Building Symbulation and running the tests
+ 
+1.  Activate (or re-activate) the virtualenv (necessary only for
+    building documentation):
+ 
+        source third-party/env/bin/activate
+ 
+    You can run this many times without any ill effects.
+ 
+    (This puts you in the development environment.)
+ 
+2.  Run the tests:
+ 
+        make test
+ 
+Congratulations! You're ready to develop!
+ 
+## Claiming an issue and starting to develop
+ 
+1.  Find an open issue and claim it.
+ 
+    Once you've found an issue you like, make sure that no one has been
+    assigned to it (see "assignee", bottom right near
+    "notifications"). Then, add a comment "I am working on this
+    issue." You've staked your claim!
+ 
+    (We're trying to avoid having multiple people working on the same
+    issue.)
+ 
+2.  In your local copy of the source code, update your master branch
+    from the main Symbulation master branch:
+ 
+        git checkout master
+        git pull upstream master
+ 
+    (This pulls in all of the latest changes from whatever we've been
+    doing on Symbulation.)
+ 
+    It is possible that when you do a [git pull]{.title-ref} you will
+    get a "merge conflict" -- This is what happens when something
+    changed in the branch you're pulling in in the same place you made
+    a change in your local copy.
+ 
+    Git will complain loudly about merges and tell you specifically in
+    which files they occurred. If you open the file, you'll see
+    something vaguely like this in the place where the merge occurred:
+ 
+        <<<<<<< HEAD
+        Changes made on the branch that is being merged into. In most cases,
+        this is the branch that you have currently checked out
+        =======
+        Changes made on the branch that is being merged in, almost certainly
+        master.
+        >>>>>>> abcde1234
+ 
+    Though there are a variety of tools to assist with resolving merge
+    conflicts they can be quite complicated at first glance and it is
+    usually easy enough to manually resolve the conflict.
+ 
+    To resolve the conflict you simply have to manually 'meld' the
+    changes together and remove the merge markers.
+ 
+    After this you'll have to add and commit the merge just like any
+    other set of changes. It's also recommended that you run tests.
```

Comment:
Instead of 'recommended' say something stronger like "You should also run the tests to make sure that everything works as expected."